packages/frontend/utils/getLocalProvider.ts

```diff
@@ -0,0 +1,17 @@
+import { Chain } from "wagmi";
```

Comment:
Love the idea of this util.

Can we move it under `utils/scaffold-eth/getLocalProvider`? Trying to get all components / hooks / utils under the "scaffold-eth" folder.